astropy/coordinates/baseframe.py

```diff
@@ -1260,17 +1260,18 @@ def is_transformable_to(self, new_frame):
         transformation between two objects of the same frame class but with
         different attributes.
         """
-
         new_frame_cls = new_frame if inspect.isclass(new_frame) else new_frame.__class__
         trans = frame_transform_graph.get_transform(self.__class__, new_frame_cls)
 
         if trans is None:
             if new_frame_cls is self.__class__:
-                return 'same'
+                transformable = 'same'
```

Comment:
I guess not anymore. There was a single-line ternary replacing this whole block... 😆

Reply:
But, does it need fixing? It is lower McCabe.

Reply:
I think in case of doubt, leave untouched. In this case, every branch has a return statement, so I don't really see a benefit to assigning to a variable and then returning that.